apply-personal-number.md

```diff
@@ -0,0 +1,31 @@
+# How to apply personal number
+## Step 1:
+##### At first check your map, search a nearest “tax agency”, “tax office” or ”Skattekontoret” (Swedish).
+
+## Step 2：
+##### Bring your Passport, Work permit card; Marriage certificate & legalized translation if not single, Kid's birth certificate & legalized translation if have kids to Skattekontoret. Take a ticket and wait.
+
+## Step 3:
+##### The officer will check your files and you can fill the website on the computer they will find for you. Fill it step by step.
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/1.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/2.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/3.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/4.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/5.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/6.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/7.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/8.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/9.png)
+
+##### After 1 to 18 months, check the mailbox, and a letter will be sent to you with a personal number and a website. Send the personal number to Ola and cc to CJ by E-mail. 
```

Comment:
Usually the waiting time is 1 to 3 months, check your mailbox regularly, Tax Agency will send you a letter by post with your personal number. When you recieve the letter, send your personal number to admin@orientalent.se to update your infomation in the company.